doc/developers/contributing.rst

```diff
@@ -394,7 +394,7 @@ complies with the following rules before marking a PR as ``[MRG]``. The
    `sklearn/linear_model/logistic.py`, running the following commands will
```

Comment:
```suggestion
   `sklearn/linear_model/_logistic.py`, running the following commands will
```

Reply:
Hi, @jeremiedbb I have changed on both places can you review them now. Thank you